dex-it/src/test/scala/com/wavesplatform/it/sync/MatcherRestartTestSuite.scala

```diff
@@ -1,9 +1,11 @@
 package com.wavesplatform.it.sync
 
+import com.github.ghik.silencer.silent
+import com.wavesplatform.dex.api.ApiOrderStatus.Status
 import com.wavesplatform.dex.domain.order.{Order, OrderType}
-import com.wavesplatform.dex.it.api.responses.dex.OrderStatus
 import com.wavesplatform.it.MatcherSuiteBase
 
+@silent("deprecated")
```

Comment:
You're right, but we use deprecated `status` or smth like that in this test suite. So builds fail in these suites because of warnings. True for all the cases below

Reply:
@deprecated annotations removed